go/libkb/globals.go

```diff
@@ -441,7 +443,7 @@ type Contexitifier interface {
 }
 
 func (g *GlobalContext) GetConfiguredAccounts() ([]keybase1.ConfiguredAccount, error) {
-	return GetConfiguredAccounts(g)
+	return GetConfiguredAccounts(g, g.SecretStore)
```

Comment:
Do we need this?  Can `GetConfiguredAccounts()` be on `SecretStore`?  So then it would be `e.G().SecretStore.GetConfiguredAccounts()`.